webui/react/src/components/UserSettings.tsx

```diff
@@ -346,33 +348,37 @@ const UserSettings: React.FC<Props> = ({ show, onClose }: Props) => {
               </div>
             </Section>
             <Section title="Advanced">
-              <Body>
-                Advanced features are potentially dangerous and could require you to completely
-                reset your user settings if you make a mistake.
-              </Body>
-              <Accordion title="I know what I'm doing">
-                <Space>
-                  <Button
-                    danger
-                    type="primary"
-                    onClick={() =>
-                      confirm({
-                        content:
-                          'Are you sure you want to reset all user settings to their default values?',
-                        onConfirm: () => {
-                          setMode(Mode.System);
-                          userSettings.clear();
-                        },
-                        onError: handleError,
-                        title: 'Reset User Settings',
-                      })
-                    }>
-                    Reset to Default
-                  </Button>
-                  <Button onClick={() => UserSettingsModal.open()}>Edit Raw Settings (JSON)</Button>
-                  <UserSettingsModal.Component />
-                </Space>
-              </Accordion>
+              <Grid gap={ShirtSize.Medium} minItemWidth={400}>
+                <Body>
+                  Advanced features are potentially dangerous and could require you to completely
+                  reset your user settings if you make a mistake.
+                </Body>
+                <Accordion title="I know what I'm doing">
+                  <Space>
```

Comment:
should be handled by task to remove antd anyway, but prefer hew `Column` to antd `Space`